### ISSUE TITLE
TL Pane in Multiview is cut off or not shown

### ISSUE THREAD

**Issue by Vesperindustrial** [HolodexNet/Holodex#305]
As of the latest update I think or at least the last few days, the TL pane in multiview is either cut off or not visible. This happens with either current livestreams or archives loading the saved TLs

**Comment by sphinxrave** [comment]
Hey, thanks for reaching out! We're having trouble replicating your problem. Please provide the following information

> A screenshot of the error

> How you got to the bug state

> Your browser version / OS

**Comment by Vesperindustrial** [comment]
Attached a pair of screesnhots, the one on the left is the comparison for the archive version. The steps I did were:
- Open multiview
- Add 2 videos as shown in the first screenshot
- toggled the TL pane in place of the chat

I talked to ricecakes about it in the Holodex Discord, neither of us could replicate it for the livestreams, it was only happening for occasional ones. The one I recall it happening most recently when it was live was Korone's mario maker stream when it was live (https://holodex.net/watch/OfQvqglpOOY). The archive TL pane not working is consistent for all archives however.

![image](https://user-images.githubusercontent.com/11395862/124999890-567a0a00-e00c-11eb-80f4-ff89ae0afcf2.png)
![image](https://user-images.githubusercontent.com/11395862/124999901-5aa62780-e00c-11eb-8673-8c10dff1f3a4.png)
